SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -0,0 +1,226 @@
+# This spec file assumes you're building on an environment where:
```

Comment:
General thought: Let's try to align this as much as possible with the cloud-hypervisor proper package.